src/pages/AdminManageUsers.js

```diff
@@ -200,18 +200,9 @@ class AdminManageUsers extends Component {
 
                     a({
                       id: user.dacUserId + "_btnResearcherReview", name: "btn_researcherReview", onClick: () => this.openResearcherReview(user.dacUserId),
-                      isRendered: user.researcher !== false && user.completed === true, className: "admin-manage-buttons col-lg-10 col-md-10 col-sm-10 col-xs-9"
+                      isRendered: true, className: "admin-manage-buttons col-lg-10 col-md-10 col-sm-10 col-xs-9"
```

Comment:
You can remove the `isRendered: true, ` as that is the default.
```suggestion
                      className: "admin-manage-buttons col-lg-10 col-md-10 col-sm-10 col-xs-9"
```